### PR TITLE
12: CI Pipeline

### PR DESCRIPTION
**Pull request by JoshLarouche**
[12](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-registration/12)

### CODE REVIEW

File: bc_obps/Dockerfile [hunk 1]
```diff
@@ -20,6 +20,8 @@ RUN useradd -ms /bin/bash -d ${HOME} --uid ${USER_ID} -g root bc_obps
 
 COPY ./ ${HOME}/
 
+RUN ls
```
Comment (andrea-williams) [inline bc_obps/Dockerfile:23]: Do we still need this line? I'm guessing it was added during debugging...

File: docs/developer-environment-setup.md [hunk 2]
```diff
@@ -61,8 +61,24 @@ You can start editing the page by modifying `pages/index.tsx`. The page auto-upd
...
+```bash
+yarn lint && yarn format
+```
 
 ## Commit Message Conventions
 
-TODO: Add commit message conventions
+We use [gitlint](https://jorisroovers.com/gitlint/) to check commit message formatting. You can enable it by using `pre-commit install --hook-type commit-msg`.
+
+This project follows the commit message conventions outlined by [Convential Commits](https://www.conventionalcommits.org/). Besides the standard commit types (message prefixes) **feat** and **fix**, we use some other types described there based on the Angular convention; some common ones among those are **test**, **docs**, **chore** and **refactor**. You can find the configuration details in the [.gitlint](../.gitlint) file
```
Comment (andrea-williams) [inline docs/developer-environment-setup.md:82]:
```suggestion
This project follows the commit message conventions outlined by [Conventional Commits](https://www.conventionalcommits.org/). Besides the standard commit types (message prefixes) **feat** and **fix**, we use some other types described there based on the Angular convention; some common ones among those are **test**, **docs**, **chore** and **refactor**. You can find the configuration details in the [.gitlint](../.gitlint) file
```

**Review by andrea-williams — APPROVED** [review]
Didn't give this a super-thorough review, but it's looking good so far! Just added a couple of comments for trivial things

File: bc_obps/Dockerfile (hunk 1 above)
Comment (Sepehr-Sobhani) [reply bc_obps/Dockerfile:23]: 💯%✅

**Pull request merged**